content/courses/glass-training-material.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "The GLaSS Training Material Builds on the Global Lakes Use Cases"
+image: "/images/courses/CarwH7tWIAAn2ry.jpg-large.webp"
+weight: 10
+draft: false
+---
+
+**Course Duration:** N/A  
```

Comment:
maybe course duration and instructor could be part of the info in the frontmatter ?